### DISCUSSION TITLE
制限数-1回以上スキルの書を使っていると解除Mを使ったときにバグる

### DISCUSSION THREAD

**Discussion by Isaac45132** [ngmansion/FE8N#93] [Q&A] [answered]
制限数が6なら5回以上書を使っているとバグります。

**Comment by aera-ngmansion** [comment]
まだバグがありましたか。その辺は複雑にしすぎてだいぶ苦労して作った所ですからねぇ…

**Comment by aera-ngmansion** [comment]
冷静になって考え直したのですけれど、そもそも[スキル数は最大5](https://github.com/ngmansion/FE8N/wiki/スキル)なので、
6以上にしてる時点で正しく動かなくても問題ないのでは？

**Reply by Isaac45132** [reply]
スキル最大数を例えば4にした場合なら3回以上スキルの書を使っていると解除mを使った時にバグります

**Reply by aera-ngmansion** [reply]
5以下でも再現するという事ですね

**Comment by aera-ngmansion** [comment]
スキル3個目以降を試したのですが、何か変な挙動してますね。
#46 の時の対応漏れがあった感じだろうかな・・・

**Reply by aera-ngmansion** [reply]
スキル書のビット演算をミスってますね。間違いない。なんだこれは・・・たまげたなあ

**Comment by aera-ngmansion** [comment, marked as answer]
#94 作成
